lib/child-writer/renderer-utils.js

```diff
@@ -43,8 +43,5 @@ exports.getApplicationEntryPoint = () => {
   const ENTRY_POINT_ARGV_INDEX = 1
   const relativeEntryPoint = electron.remote.process.argv[ENTRY_POINT_ARGV_INDEX]
 
-  // On GNU/Linux, `pkexec` resolves relative paths
-  // from `/root`, therefore we pass an absolute path,
-  // in order to be on the safe side.
```

Comment:
I've not been following the progress of this PR, but why has this comment been removed?

Reply:
We don't use `pkexec` anymore so this explanation isn't necessary